packages/ui/src/modal/buy/BuyModal.tsx

```diff
@@ -242,6 +242,7 @@ export function BuyModal({
               <Flex direction="column">
                 <TokenLineItem
                   chainId={modalChain?.id}
+                  chainName={modalChain?.name}
```

Comment:
maybe not worth changing at this point but wonder if these should be taking chain param that's our Reservoir Chain

Reply:
Yeah good call. Will do 👍